indico/modules/events/management/settings.py

```diff
@@ -22,3 +23,7 @@
     'privacy_policy_urls': [],
     'privacy_policy': '',
 })
+
+event_settings = SettingsProxy('events', {
```

Comment:
Let's call this variable `global_event_settings` since we already use `event_settings` quite a lot in other places, in particular when it comes to plugins which have an `event_settings` property for event-*specific* settings.

Reply:
Done.